published-namespaces.txt

```diff
@@ -159,16 +159,20 @@ AndroidX.Core.Net
 AndroidX.Core.OS
 AndroidX.Core.Provider
 AndroidX.Core.Role
+AndroidX.Core.Service.QuickSettings
 AndroidX.Core.SplashScreen
 AndroidX.Core.Telephony
 AndroidX.Core.Telephony.Mbms
 AndroidX.Core.Text
+AndroidX.Core.Text.Method
 AndroidX.Core.Text.Util
 AndroidX.Core.Transition
 AndroidX.Core.Util
 AndroidX.Core.View
 AndroidX.Core.View.Accessibility
 AndroidX.Core.View.Animation
+AndroidX.Core.View.AutoFill
+AndroidX.Core.View.Contentcapture
```

Comment:
`ContentCapture` is a compound word:

https://developer.android.com/reference/androidx/core/view/contentcapture/ContentCaptureSessionCompat?hl=en

Reply:
missed that one

added to Directory.Build.props

Reply:
fixed